.github/workflows/cli-build-instructions.yml

```diff
@@ -14,12 +14,31 @@ env:
 
 jobs:
 
+  # Using the Java style formatter google-java-style provided by the Spotless
+  # plugin configured in the root pom.xml using 4-space indents (AOSP style).
+  # Spotless is configured to run only on files in this branch (PR) that differ
+  # from origin/main
+  formatter:
+    name: Formatter + Style checker
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Unicode Tools
+        uses: actions/checkout@v2
+      # Should we use "upstream" instead of "origin" in Spotless configuration
+      # for the git remote name for unicode-org/unicodetools?
+      - name: Fetch git branches for unicode-org/unicodetools
+        run: |
+          git fetch origin
```

Comment:
or is this a different fetch than the PR's fetch?

Reply:
Done. Thanks for pointing out the `fetch-depth` config for `actions/checkout`, I'm using that instead.  The reason for having the `git fetch` in the first place is because of the Spotless configuration for the "ratchet" feature that only checks formatting diffs to modified files in the PR (and it needs the reference point `origin/main` to diff and grok what's in the PR).... however, we are leaning towards having a single big bang change globally for formatting, which will remove noise from future PRs by allowing those future PRs to only spotlight the changed lines of interest.  That makes sense, and would obviate the need for the ratchet feature.